USAGEGUIDE.md

```diff
@@ -5,80 +5,79 @@
 
 ## Table of Contents
 
-- [Prerequisites](#prerequisites)
-- [Deploying 1Password Connect to Kubernetes](#deploying-1password-connect-to-kubernetes)
-- [Kubernetes Operator Deployment](#kubernetes-operator-deployment)
-- [Usage](#usage)
-- [Configuring Automatic Rolling Restarts of Deployments](#configuring-automatic-rolling-restarts-of-deployments)
-- [Development](#development)
+1. [Prerequisites](#prerequisites)
+2. [Configuration Options](#configuration-options)
+3. [Use Kubernetes Operator with Service Account](#use-kubernetes-operator-with-service-account)
+    - [Create a Service Account](#1-create-a-service-account)
+    - [Create a Kubernetes secret](#2-create-a-kubernetes-secret-for-the-service-account)
+    - [Deploy the Operator](#3-deploy-the-operator)
+4. [Use Kubernetes Operator with Connect](#use-kubernetes-operator-with-connect)
+    - [Deploy with Helm](#1-deploy-with-helm)
+    - [Deploy manually](#2-deploy-manually)
+5. [Logging level](#logging-level)
+6. [Usage examples](#usage-examples)
+7. [How 1Password Items Map to Kubernetes Secrets](#how-1password-items-map-to-kubernetes-secrets)
+8. [Configuring Automatic Rolling Restarts of Deployments](#configuring-automatic-rolling-restarts-of-deployments)
+9. [Development](#development)
+
+---
 
 ## Prerequisites
 
 - [1Password Command Line Tool Installed](https://1password.com/downloads/command-line/)
```

Comment:
Where is the CLI needed? Is it only needed for Connect deployments?

Reply:
As now we refer users to the dev portal for the instructions of how to deploy operator with connect, in this doc we never mention CLI, so we can remove it.

Thinking more about this, as now this section will have only `kubectl` and `docker`, I think we can remove it completely. I believe users who wants to use Operator already know that they need `kubeclt` and `docker` as it is intuitive.

So, I'll remove this section completely. We'll see how it goes, and if something we can put it back.